flow/Platform.actor.cpp

```diff
@@ -4104,53 +4097,53 @@ void platformSpecificDirectoryOpsTests(const std::string& cwd, int& errors) {
 	ASSERT(symlink("../backups/four", "simfdb/backups/five") == 0);
 
 	errors += testPathFunction2(
```

Comment:
this is complex, same function with differente params, better put the params in an approriate data structure and iterate, doing append.

Reply:
I agree, I think that this would be best. I was hoping to correct the `std::filesystem` but there's still a lot to go through then go back to refactor this as a separate PR. Thank you for the heads up, I'll note this down for future PR to refactor this